tests/Php84/Php84Test.php

```diff
@@ -256,6 +256,16 @@ public function testMbTrimCharactersEncoding()
         mb_internal_encoding($old);
     }
 
+    /**
+     * @covers \Symfony\Polyfill\Php84\Php84::bcdivmod
+     *
+     * @dataProvider bcDivModProvider
+     */
+    public function testBcDivMod(string $num1, string $num2, ?int $scale = null, array $expected): void
```

Comment:
this test needs `@requires extension bcmath` as the implementation would break without that extension

Reply:
Brilliant, thank you.